### PR TITLE
refactor: change parameter exclude unmerged to include

### LINKED ISSUE #170: Include unmerged reads
### Description of the bug

I think this check in the pipeline (`taxprofiler.nf:31`):

```nextflow
if (params.shortread_qc_excludeunmerged && !params.shortread_qc_mergepairs) exit 1, "ERROR: [nf-core/taxprofiler] cannot include unmerged reads when merging not turned on. Please specify --shortread_qc_mergepairs"
```

shows that we should maybe rename the parameter to `shortread_qc_includeunmerged` and then also change the logic in `modules.config:82` for fastp:

```nextflow
params.shortread_qc_includeunmerged ? "--include_unmerged" : '',
```

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by Midnighter**
## PR checklist

- [x] fix #170 
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -125,11 +125,11 @@
...
+                "shortread_qc_includeunmerged": {
                     "type": "boolean",
                     "fa_icon": "far fa-times-circle",
-                    "description": "Discard unmerged reads from paired-end merging",
-                    "help_text": "Turns off the inclusion of unmerged reads in resulting processing FASTQ file of paired-end sequencing data when using `fastp`.\n\nThis can be useful in cases where you prefer to have very short reads (e.g. aDNA), thus excluding longer-reads or possibly faulty reads where one of the pair was discarded.\n\n> Modifies tool parameter(s):\n> - removed from reads `--include_unmerged`\n"
+                    "description": "Include unmerged reads from paired-end merging in the downstream processing",
+                    "help_text": "Turns on the inclusion of unmerged reads in resulting processing FASTQ file of paired-end sequencing data when using `fastp`.\n\nExcluding unmerged reads can be useful in cases where you prefer to have very short reads (e.g. aDNA), thus excluding longer-reads or possibly faulty reads where one of the pair was discarded.\n\n> Modifies tool parameter(s):\n> - adds option `--include_unmerged`\n"
```
Comment (jfy133) [inline nextflow_schema.json:132]: Should mention this applies to AdapterRemoval too (influencing what output files are mixed into the final output FASTQ).

The end help text here should be something like:

> Modifies fastp parameter(s): `--<whatever it is>`

**Review by jfy133 — CHANGES_REQUESTED** [review]
Overall looks good, one comment on docs

File: nextflow_schema.json (hunk 1 above)
Comment (Midnighter) [reply nextflow_schema.json]: I've added some words on AdapterRemoval. Let me know what you think.

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
@Midnighter merge away :rocket:

**Pull request merged**